crates/sui-network/src/discovery/server.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{Discovery, NodeInfo, State};
+use anemo::{Request, Response};
+use serde::{Deserialize, Serialize};
+use std::sync::{Arc, RwLock};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct GetKnownPeersResponse {
+    pub own_info: NodeInfo,
+    pub known_peers: Vec<NodeInfo>,
+}
+
+pub(super) struct Server {
```

Comment:
Andrew's favorite topic, consistent naming

in narwhal these are called handler (eg `PrimaryReceiverHandler`, `WorkerReceiverHandler`). do you think the ones in narwhal should be renamed server, this should be renamed handler, or if neither - why neither? :)

Reply:
So the great thing about namespaces is that you don't really need to name this this `DiscoveryServer` or `DiscoveryHandler` because the namespace tells you that info! This type is completely private to the discovery module though so the naming isn't a huge issue either way

Reply:
Agree on the prefixes being unnecessar, but we should still be consistent between `Server` vs `Handler` for anemo method handler impl structs?